PyU4V/tools/openstack/migrate_utils.py

```diff
@@ -0,0 +1,911 @@
+# Copyright (c) 2019 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+OpenStack migrate_utils.py.
+
+Migrate utilities for OpenStack
+"""
+
+from __future__ import print_function
+
+from builtins import input
+import logging
+import re
+import sys
+
+from PyU4V.utils import exception
+
+import prettytable
```

Comment:
Pretty Table will need listed in setup.py as a package dependency and also in our docs as a requirement because it isn't  a native Python library